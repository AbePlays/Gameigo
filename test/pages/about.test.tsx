import AboutPage from 'pages/about';
import { render, screen } from '../test-utils';

describe('Testing About Page', () => {
  it('should render about page', () => {
    render(<AboutPage />);
    expect(screen).toMatchSnapshot();
  });
});
