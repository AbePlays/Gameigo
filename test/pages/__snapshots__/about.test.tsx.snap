// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing About Page should render about page 1`] = `
<div>
  <div
    class="css-1fw334b"
    style="opacity: 0; transform: translateY(20px) translateZ(0);"
  >
    <main>
      <div
        class="chakra-stack css-1cuod9q"
      >
        <div
          class="css-0"
          style="opacity: 0; transform: translateY(50px) translateZ(0);"
        >
          <h2
            class="chakra-heading css-65c5cv"
          >
            Small and Crafty
          </h2>
          <div
            aria-hidden="true"
            class="css-1asknlk"
          />
          <p
            class="chakra-text css-1dtrxbm"
            style="text-align-last: center;"
          >
            Gameigo is a video game discovery platform where you can keep all your games in one unified profile and create your exclusive collection. The application is powered by a public API provided by RAWG which houses more than 350,000 games across half a hundred platforms.
          </p>
        </div>
        <div
          class="css-0"
          style="opacity: 0; transform: translateY(50px) translateZ(0);"
        >
          <h2
            class="chakra-heading css-65c5cv"
          >
            Creator
          </h2>
          <div
            aria-hidden="true"
            class="css-1asknlk"
          />
          <p
            class="chakra-text css-1dtrxbm"
            style="text-align-last: center;"
          >
            Hi there, I'm Abhishek - aka Abe ðŸ‘‹ I am a self learner trying to get hands on new technologies and producing exciting products that are as smart, as they are effective. Wanna talk about Computer Sciency things? Drop a message : )
          </p>
        </div>
        <div
          class="css-0"
          style="opacity: 0; transform: translateY(50px) translateZ(0);"
        >
          <h2
            class="chakra-heading css-65c5cv"
          >
            Contact
          </h2>
          <div
            aria-hidden="true"
            class="css-1asknlk"
          />
          <div
            class="chakra-stack css-1f49ad2"
          >
            <a
              class="chakra-link css-1fl33w"
              href="https://github.com/AbePlays"
              target="_blank"
            >
              Github
            </a>
            <a
              class="chakra-link css-1fl33w"
              href="mailto:abhi.rawat456@gmail.com"
            >
              Mail
            </a>
            <a
              class="chakra-link css-1fl33w"
              href="https://www.linkedin.com/in/abe10/"
              target="_blank"
            >
              LinkedIn
            </a>
          </div>
        </div>
      </div>
    </main>
  </div>
  <span
    hidden=""
    id="__chakra_env"
  />
</div>
`;
